flink-staging/flink-gelly-scala/src/test/scala/org/apache/flink/graph/scala/test/operations/MapEdgesITCase.scala

```diff
@@ -92,6 +92,30 @@ MultipleProgramsTestBase(mode) {
       "5,1,52\n"
   }
 
+  @Test
+  @throws(classOf[Exception])
+  def testWithSameValueAlgorithm {
+    val env: ExecutionEnvironment = ExecutionEnvironment.getExecutionEnvironment
+    val graph: Graph[Long, Long, Long] = Graph.fromDataSet(TestGraphUtils
+      .getLongLongVertexData(env), TestGraphUtils.getLongLongEdgeData(env), env)
+
+   class SameValueAlgorithm extends GraphAlgorithm[Long, Long, Long] {
+     override def run(graph: Graph[Long, Long, Long]): Graph[Long, Long, Long] = {
+       graph.mapEdges(edge => edge.getValue + 1)
+     }
+   }
+
+    graph.run(new SameValueAlgorithm).getEdgesAsTuple3().writeAsCsv(resultPath)
```

Comment:
If I'm not mistaken, then writing to files while running on Travis can cause sometimes test failures. Therefore, we've started using the `collect` method and then to compare the results in your case with `compareResultAsTuples`. That way, we only work on data kept in memory. In the current master version of the `MapEdgesITCase` this is already the case. Maybe you haven't rebased on the latest master?